docs/source/reference/cli.rst

```diff
@@ -28,6 +28,19 @@ page.
     -s STYLE_FILE, --style-file STYLE_FILE
                             colour bar definition to use
 
+
+The ``cset bake`` commands need to be run out of the ``CSET/recipe`` folder.
```

Comment:
I think our recommendation should rather be for people to use the `cset cookbook` command to get the recipes they need.